api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -16,8 +16,8 @@
     liquibase-update-to-2.0.xml, which have been used for older releases.
 -->
 <databaseChangeLog logicalFilePath="liquibase-update-to-latest.xml" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
-	xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd
```

Comment:
Did you make any changes here?

Reply:
No I did not. This could have been as a result of resolving merge conflicts

Reply:
Ok. revert it.